### ISSUE TITLE
API to get currently set timeouts

### ISSUE THREAD

**Issue by black-snow** [SeleniumHQ/selenium#8786]
## 🚀 Feature Proposal

Three new methods for `org.openqa.selenium.WebDriver.Timeouts`:

```java
long implicitlyWait();
long scriptTimeout();
long pageLoadTimeout();
```

or alternatively

```java
long getImplicitlyWait();
long getScriptTimeout();
long getPageLoadTimeout();
```

returning the values set via their counterpart methods.

## Motivation

I've encountered a situation where I'd like to temporarily change the timeouts set via 
`driver.manage().timeouts()`. In order to do this I have to retrieve and store the currently set values for those timeouts.

## Example

Somewhere early:

```java
driver.manage().timeouts().implicitlyWait(10L, TimeUnit.SECONDS);
```

Now there's a place where I loop over some WebElements and check for some children but I do not want to implicitly wait 10 seconds every time:

```java
final long previousWait = driver.manage().timeouts().implicitlyWait();
driver.manage().timeouts().implicitlyWait(1L, TimeUnit.SECONDS);
for (final WebElement row : getResults()) {
    // do stuff
}
driver.manage().timeouts().implicitlyWait(previousWait);
```

**Comment by nitesh261193** [comment]
Hi ,
Instead of using implicit wait , fluent wait is much better option to choose. 
public void fluentWait(int timeout){
     Wait wait = new FluentWait(WebDriver reference)
     .withTimeout(timeout, SECONDS)
     .pollingEvery(timeout, SECONDS)
     .ignoring(Exception.class);
}
timeout - max time to wait
pooling time- It breaks total timeout into small time cycle. condition will check after cycle. if condition get true, wait method will
stop

**Comment by diemol** [comment]
@black-snow, I believe @nitesh261193's comment makes sense, the use case you describe is the reason why explicit and fluent waits exist. Basically, you'd like to have an implicit wait behaving differently on each part of the code, that is where an explicit/fluent wait can help.

**Comment by black-snow** [comment]
Thanks for your responses. I know and use fluent waits but you don't always have full control over the code and when you are able to set those things you should probably also be able to retrieve the current values. Imagine a lib doing Selenium stuff but you want to know the implicit timeouts. Or you recognize some network congestion and want to temporarily increase timeouts.

**Comment by gauravrajput44** [comment]
@black-snow I think if you are using a lib then I would suggest you to override that class. 
Selenium is already providing all sort of different waits. 
Personally I do not favor implicit wait and suggest the same to you.

**Comment by barancev** [comment]
Implemented by commit 2a5118b57d977be219df525e5e122bdd4963ac99
